modules/installation-configuration-parameters.adoc

```diff
@@ -1150,6 +1158,10 @@ at least two zones.
 |The name of the Azure cloud environment that is used to configure the Azure SDK with the appropriate Azure API endpoints. If empty, the default value `AzurePublicCloud` is used.
 |Any valid cloud environment, such as `AzurePublicCloud` or `AzureUSGovernmentCloud`.
 
+|`platform.azure.defaultMachinePlatform.vmNetworkingType`
+|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance.
+|`Accelerated` or `Basic`. If instance type of control plane and compute machines support `Accelerated` networking, installer by default enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```

Comment:
```suggestion
|`Accelerated` or `Basic`. If instance type of control plane and compute machines support `Accelerated` networking, by default, the installer enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```